tests/test_lightning.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for training with PyTorch Lightning."""
+import itertools
+import unittest
+
+import pytest
+
+from pykeen.models import model_resolver
+
+try:
+    import pytorch_lightning
+
+    from pykeen.contrib.lightning import lit_module_resolver
+
+    LIT_MODULES = lit_module_resolver.lookup_dict.keys()
+except ImportError:
+    pytorch_lightning = None
+    lit_module_resolver = None
+    LIT_MODULES = []
+
+INTERACTIONS = model_resolver.lookup_dict.keys()
+
+
+@unittest.skipIf(
+    pytorch_lightning is None, reason="PyTorch Lightning tests require `pytorch-lightning` to be installed"
```

Comment:
this isn't such a heavy/controversial dependency so I'd suggest assuming it's available

Reply:
926fc863